sphinx_immaterial/content_tabs.py

```diff
@@ -123,23 +122,26 @@ def depart_tab_label(self, node):
 
 def visit_tab_set(self: HTMLTranslator, node: content_tab_set):
     # increment tab set counter
-    self.tab_set_count = getattr(self, "tab_set_count", 0) + 1
+    self.tab_set_count = getattr(self, "tab_set_count", 0) + 1  # type: ignore
```

Comment:
This will always be an `int`.
```suggestion
    self.tab_set_count: int = getattr(self, "tab_set_count", 0) + 1  # type: ignore
```

Reply:
damn you're fast. I noticed and fixed my suggestion after you commit it. 😆